Dockerfile.certgen

```diff
@@ -1,6 +1,8 @@
 FROM alpine
 
 RUN apk add openssl
+RUN mkdir -p /etc/ssl/private
+RUN mkdir -p /etc/ssl/certs
```

Comment:
What do you think about downgrading base image to `alpine:3.14` instead of creating these directories manually? My motivation is to keep it stable and avoid other breaking changes once `alpine:latest` is updated. This is rather unlikely, but maybe openssl package won't be available by default someday...